client/js/components/statement/listStatements/ListStatements.vue

```diff
@@ -21,6 +21,7 @@
         <dp-button
           class="ml-auto"
           variant="outline"
+          data-cy="listStatement:searchReset"
```

Comment:
Here and other changes: 

```suggestion
          data-cy="listStatements:searchReset"
```